From: Dmitry Atamanov <data-man@users.noreply.github.com>
Date: Sun, 26 Apr 2020 02:26:53 +0500
Subject: fix(ui): fix compilation error with Qt 5.15 (#1218)

(cherry picked from commit 064aaa05d6a3ba4ba3cf648199a3109aba2f41fe)
---
 src/libs/ui/searchitemdelegate.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libs/ui/searchitemdelegate.cpp b/src/libs/ui/searchitemdelegate.cpp
index 4bd6d0a..d8ad79b 100644
--- a/src/libs/ui/searchitemdelegate.cpp
+++ b/src/libs/ui/searchitemdelegate.cpp
@@ -27,6 +27,7 @@
 #include <QFontMetrics>
 #include <QHelpEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QToolTip>
 
 using namespace Zeal::WidgetUi;
